agent-connection/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/PythonRefAgentConnectionTest.kt

```diff
@@ -37,6 +41,9 @@ class PythonRefAgentConnectionTest {
                         throw AgentConnectionException(ex.message!!)
                     }
                     else acceptInvite(invitationString).subscribe { master ->
+                        val tails = master.requestTails(tailsHash).toBlocking().value().tails[tailsHash]
+                        if (tails != tailsHash)
+                            throw AgentConnectionException("Tails are wrong!!! hash $tailsHash, received $tails")
```

Comment:
Tails *is* wrong?

Reply:
My suggestion would be
1. Tails are wrong
2. Tail is wrong
3. Tails file content doesn't match
What would you choose?

Reply:
3rd one seems to be correct.

Reply:
done